Clients/src/presentation/components/Modals/NewVendor/index.tsx

```
@@ -14,580 +14,823 @@
-  const handleDateChange = (newDate: Dayjs | null) => {
-    setValues((prevValues) => ({
-      ...prevValues,
-      vendorDetails: {
-        ...prevValues.vendorDetails,
-        reviewDate: newDate ? newDate.toISOString() : "",
-      },
-    }));
-  };
+	const updateVendor = async (vendorId: number, updatedVendorDetails: VendorDetails) => {
+		// Make a call to backend and update the vendor'
+		debugger;
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)